packages/mesos/extra/dcos-mesos-slave.service

```diff
@@ -12,6 +12,7 @@ EnvironmentFile=/opt/mesosphere/environment
 EnvironmentFile=/opt/mesosphere/etc/mesos-slave-common
 EnvironmentFile=-/opt/mesosphere/etc/mesos-slave-common-extras
 EnvironmentFile=/opt/mesosphere/etc/mesos-slave
+EnvironmentFile=-/opt/mesosphere/etc/mesos-slave-extras
```

Comment:
`mesos-slave-common-extras` is for variables that the downstream wants to apply to both public and private slaves.
`mesos-slave-extras` is for variables the downstream only wants to apply to private slaves. Maybe we should call it `mesos-slave-private-extras`, but we haven't used "private" anywhere else before.